docs/search-configuration.md

```diff
@@ -79,3 +79,62 @@ By default the post title, excerpt, content, author name and taxonomy terms are
 ```
 
 It is important to note that the field names should match the fields in the elasticsearch index and do not always correspond 1:1 with database fields.
+
+## Fuzzy Matching
+By default some degree of fuzzy matching is allowed so that simple spelling errors can still return some results, for example "breif" would match "brief". This can sometimes result in unwanted search results however with short words and acronyms.
+
+Fuzzy matching works by providing an edit distance as an integer from 0-2. The number indicates how many edits are allowed for a term to match. Edits can be one of the following:
+
+* Changing a character (box → fox)
+* Removing a character (click → lick)
+* Inserting a character (sic → sick)
+* Transposing two adjacent characters (act → cat)
+
+**Note:** when using "advanced" search mode quoted strings will not use fuzzy matching.
+
+**Note:** if you have synonyms configured they will not use fuzzy matching.
+
+Below is the default fuzzy search configuration.
+
+```json
+{
+	"extra": {
+		"altis": {
+			"modules": {
+				"search": {
+					"fuzziness": {
+						"distance": "auto:4,7",
+						"prefix-length": 1,
+						"max-expansions": 40,
+						"transpositions": true
+					}
+				}
+			}
+		}
+	}
+}
+```
+
+**`distance`**
+
+This determines the edit distance for fuzzy matching. This value can also be set as the value for `fuzziness` directly if you do not need to edit the other options.
+
+- If an integer is provided this is used as a fixed edit distance eg. 0, 1 or 2 edits allowed
+- If a string is provided it must be in the form `auto:[min],[max]`
```

Comment:
Are we using the `auto:` prefix because it's part of ES or something else? Is it just future-proofing?

Reply:
Part of ES https://www.elastic.co/guide/en/elasticsearch/reference/6.8/common-options.html#fuzziness

Reply:
I toyed with some other formats but it felt like an unnecessary abstraction at that point.

Reply:
👍 can we link the ES docs directly from here so it's clearer that you're setting ES config? Will help with being able to explore how to tune the settings I think.